Makefile

```diff
@@ -81,7 +81,8 @@ GINKGO_FOCUS ?=
 GINKGO_SKIP ?=
 GINKGO_LABEL_FILTER ?=
 GINKGO_NODES ?= 1
-GINKGO_TIMEOUT ?= 2h
+GINKGO_TIMEOUT ?= 3h
```

Comment:
We're already pretty close to the 2h timeout. I want to avoid getting test failures if we reach the 2h timeout sometimes